lib/js/components/Form/InputField/useInputFieldWithinForm.ts

```diff
@@ -1,37 +1,12 @@
-import { computed, inject, MaybeRefOrGetter, ref, Ref, toValue } from 'vue';
-import { FormContextKey, useField } from 'vee-validate';
+import { MaybeRefOrGetter, ref, Ref } from 'vue';
+import { useFormFieldWithinForm } from '../../../composables/useFormFieldWithinForm';
 
 export function useInputFieldWithinForm(
 	name: MaybeRefOrGetter<string | undefined>,
 	modelValue: Ref<string | undefined>,
 ) {
-	const form = inject(FormContextKey, null);
-	const nameValue = toValue(name);
+	const { value, errors, field } = useFormFieldWithinForm(name, modelValue);
 
-	// this computed property was created only for type safety so we can use it with `useField`
-	const nameString = computed(() => {
-		const value = toValue(name);
-
-		if (value === undefined) {
```

Comment:
in new method we skipped this check. Is it on purpose?

Reply:
Yes, it's not needed anymore as we have:
```
if (nameValue && !form) {
```
in `useFormFieldWithinForm.ts`

Reply:
`name` is only required if we want to use the field with `vee-validate` - it's not needed if we want to use `v-model` 
That previous check was added just to make TS happy - not it's handled with the new check in `nameValue ? useField<Array<string>>(nameValue, undefined) : null` - so it the name is not defned we just don't use `useField`